.github/workflows/docker_publish.yml

```diff
@@ -0,0 +1,46 @@
+
+on:
+  push:
+    branches: ['gh_action_gh_container_registry']  # todo replace with main once we know this is working
+
+jobs:
+  build-and-push-image:
+    runs-on: ubuntu-latest
+
+    strategy:
+      matrix:
+       name : [
+          pymoab-py2-18.04,
+          pymoab-py3-18.04,
+          ]
+
+    steps:
+      - name: Checkout repository
+        uses: actions/checkout@v2
+
+      - name: Set up QEMU
+        uses: docker/setup-qemu-action@v1
+
+      - name: Log in to the Container registry
+        uses: docker/login-action@v1
+        with:
+          registry: ghcr.io
+          username: ${{ github.repository_owner }}
+          password: ${{ secrets.GITHUB_TOKEN }}
+
+      - name: Build and push pymoab Docker image
+        uses: docker/build-push-action@v2
+        with:
+          file: ${{ matrix.name }}/Dockerfile
+          context: .
+          push: true
+          tags: ghcr.io/svalinn/${{ matrix.name }}
+
+        if: success()
+      - name: Build and push visit Docker image
+        uses: docker/build-push-action@v2
+        with:
+          file: pymoab-visit-py2-18.04/Dockerfile
```

Comment:
Can we assign `pymoab-visit-py2-18.04` to a variable and reuse it in the filename and tags?